src/main/java/com/lorepo/icplayer/client/module/text/TextPrintable.java

```diff
@@ -37,6 +44,19 @@ public String getPrintableHTML(String className, boolean showAnswers) {
 		result = PrintableContentParser.addClassToPrintableModule(result, className, !model.isSplitInPrintBlocked());
 		return result;
 	}
+
+	private Map<String, String> getAnswersFromPrintableState() {

```

Comment:
Skoro mamy obiekt ktory ogarnia stan - to czemu ten obiekt nie zwraca nam stworzonej mapy lub nulla po prostu w swojej funkcji tylko tutaj to inicjalizujemy?